app/workers/future_learn_course_worker.rb

```diff
@@ -0,0 +1,101 @@
+# -*- encoding : utf-8 -*-
```

Comment:
The magic comment for the encoding has changed and the second magic comment is missing.